django-stubs/core/paginator.pyi

```diff
@@ -19,6 +20,8 @@ class _SupportsPagination(Protocol[_T], Sized, Iterable):
 
 class Paginator(Generic[_T]):
     ELLIPSIS: ClassVar[_StrPromise]
+    default_error_messages: ClassVar[_ErrorMessagesDict]
```

Comment:
values are actually guaranteed to be `_StrPromise`, but I think it's fine to leave it this way (`dict[str, _StrOrPromise]`)

Reply:
I suspect you'd then run in to troubles if you want to declare your own `default_error_messages` not bothering about translation strings. e.g.

```python
class X(Paginator):
    default_error_messages = {"key": "not a translated string"}
```

Reply:
Oh yes right